### ISSUE TITLE
Jenkins: pkg_resources.DistributionNotFound: The 'PyYAML<5.1' distribution was not found and is required by ceph-salt

### ISSUE THREAD

**Issue by sebastian-philipp** [ceph/ceph-salt#125]
https://ceph-ci.suse.de/job/sesdev-integration/job/master/1/execution/node/88/log/

```
    admin: ++ ceph-salt config /Cluster/Minions add admin.mini.com
    admin: Traceback (most recent call last):
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 578, in _build_master
    admin:     ws.require(__requires__)
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 895, in require
    admin:     needed = self.resolve(parse_requirements(requirements))
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 786, in resolve
    admin:     raise VersionConflict(dist, req).with_context(dependent_req)
    admin: pkg_resources.ContextualVersionConflict: (PyYAML 5.1.2 (/usr/lib64/python3.6/site-packages), Requirement.parse('PyYAML<5.1'), {'ceph-salt'})
    admin: 
    admin: During handling of the above exception, another exception occurred:
    admin: 
    admin: Traceback (most recent call last):
    admin:   File "/usr/bin/ceph-salt", line 6, in <module>
    admin:     from pkg_resources import load_entry_point
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3123, in <module>
    admin:     @_call_aside
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3107, in _call_aside
    admin:     f(*args, **kwargs)
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3136, in _initialize_master_working_set
    admin:     working_set = WorkingSet._build_master()
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 580, in _build_master
    admin:     return cls._build_from_requirements(__requires__)
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 593, in _build_from_requirements
    admin:     dists = ws.resolve(reqs, Environment())
    admin:   File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 781, in resolve
    admin:     raise DistributionNotFound(req, requirers)
    admin: pkg_resources.DistributionNotFound: The 'PyYAML<5.1' distribution was not found and is required by ceph-salt
Command '['vagrant', 'up']' failed: ret=1 stderr:
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
```

**Comment by toabctl** [comment]
Already solved in git via https://github.com/ceph/ceph-salt/commit/a7f7499733aa57dde15dae4b87216cff5f2698cb

**Comment by sebastian-philipp** [comment]
Fixed by #117
